docs/taskruns.md

```diff
@@ -509,7 +517,7 @@ spec:
 ```
 
 
-### Debugging a `TaskRun`
+## Debugging a `TaskRun`
 
 #### Breakpoint on Failure
```

Comment:
We should probably drop the heading on this to 3 `#` to bring it in line with its changed parent heading. Same for `#### Debug Environment` below - should now be `### Debug Environment`.

Reply:
Thanks @sbwsg, you are right. I've changed the levels of the two headings.